libopenfpga/libpinconstrain/test/main.cpp

```diff
@@ -0,0 +1,30 @@
+
+/********************************************************************
+ * Unit test functions to validate the correctness of
+ * 1. parser of data structures
+ * 2. writer of data structures
+ *******************************************************************/
+/* Headers from vtrutils */
+#include "vtr_assert.h"
```

Comment:
Any suggestion to run the test here? I can create unit tests and deploy to CI

Reply:
I think we can create a pin_c run here and write out the generated I/O pin .place file.
Like the full test shows:
pin_c --pcf OpenFPGA/openfpga_flow/tasks/basic_tests/constrain_pin_location/config/and2.pcf
 --blif and2.blif
 --pinmap_xml OpenFPGA/openfpga_flow/tasks/basic_tests/constrain_pin_location/config/          pinmap_k4_N4_tileable_40nm.xml
 --csv_file OpenFPGA/openfpga_flow/tasks/basic_tests/constrain_pin_location/config/            pinmap_k4_N4_tileable_40nm.csv
 --output and2_constrain_pin.place

And then write out this and2_constrain_pin.place. Is that OK?